.github/workflows/build.yml

```diff
@@ -329,12 +344,12 @@ jobs:
     strategy:
       matrix:
         server:
-          - version: v1.15.0-dev
-            image: hashicorppreview/consul:1.15-dev
```

Comment:
1.15 will be LTS, and gets a patch next week - should probably keep for now?

Reply:
That's fair, I will update 👍